src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,28 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(actor);
+            transaction.commit();
+        } catch (RuntimeException e) {
+            transaction.rollback();
+            throw new DataProcessingException("Can't add actor to BD", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't add actor to BD " + actor, e);
```

lets make exception messages more informative